neqo-bin/src/lib.rs

```diff
@@ -222,6 +227,10 @@ impl QuicParameters {
             .max_streams(StreamType::BiDi, self.max_streams_bidi)
             .max_streams(StreamType::UniDi, self.max_streams_uni)
             .idle_timeout(Duration::from_secs(self.idle_timeout))
+            .initial_rtt(
+                self.initial_rtt_ms
+                    .map_or(INITIAL_RTT, Duration::from_millis),
```

Comment:
Now that `initial_rtt_ms` is an `Option`, this should map to `None` and not `INITIAL_RTT` when nothing is specified.